front/app/components/AuthProviders/messages.ts

```diff
@@ -84,6 +84,10 @@ export default defineMessages({
     id: 'app.containers.SignUp.viennaConsentUserName',
     defaultMessage: 'User name',
   },
+  viennaThePrivacyPolicy: {
+    id: 'app.containers.SignUp.viennaThePrivacyPolicy',
+    defaultMessage: 'the privacy policy',
```

Comment:
I'll translate it in CrowdIn for DE when this got merged.

Reply:
We usually submit our dev branch to Crowdin, translate it there, and then we merge. 

With your flow, I think there's a chance this copy will be released without the translation. But maybe it's still fine.

Reply:
We usually submit our dev branch to Crowdin, translate it there, and then we merge. 

With your flow, I think there's a chance this copy will be released without the translation. But maybe it's still fine.

Reply:
Yes I know, but it's literally a single string for a single language, I think it's easier if I just translate this directly after merge.